pydis_site/apps/api/viewsets/bot/user.py

```diff
@@ -240,10 +242,21 @@ def bulk_patch(self, request: Request) -> Response:
     def metricity_data(self, request: Request, pk: str = None) -> Response:
         """Request handler for metricity_data endpoint."""
         user = self.get_object()
+
+        try:
+            Infraction.objects.get(user__id=user.id, active=True, type="voice_ban")
+        except ObjectDoesNotExist:
+            voice_banned = False
+        except MultipleObjectsReturned:
```

Comment:
Will there ever be multiple active voice bans for a user?

These cannot be in the DB due to the one active infraction type per user constraint.

```
ERROR:  duplicate key value violates unique constraint "unique_active_infraction_per_type_per_user"
```